### DISCUSSION TITLE
Help with docker setup

### DISCUSSION THREAD

**Discussion by mbienvenu** [AykutSarac/jsoncrack.com#196] [Q&A]
Hello everyone.

Thank you @AykutSarac  and other contributors for this amazing tool.

I'm having some issues using the dockerfile on Windows.. 

Getting the error below 

 > [stage-1 3/3] COPY default.conf /etc/nginx/conf.d/default.conf:
------
failed to compute cache key: "/default.conf" not found: not found

**Comment by AykutSarac** [comment]
Dockerfile has been updated.
